srcpkgs/ta-lib/patches/ta-lib-0.4.0-asneeded.patch

```diff
@@ -0,0 +1,83 @@
+ src/Makefile.am             |    2 +-
+ src/Makefile.in             |    2 +-
+ src/ta_abstract/Makefile.am |    8 ++++----
+ src/ta_common/Makefile.am   |    6 +++---
+ src/ta_func/Makefile.am     |    6 +++---
+ 5 files changed, 12 insertions(+), 12 deletions(-)
+
+diff --git a/src/Makefile.am b/src/Makefile.am
+index 12bc5b6..b1a0c6b 100644
+--- a/src/Makefile.am
++++ b/src/Makefile.am
+@@ -8,7 +8,7 @@ libta_lib_la_SOURCES =
+ libta_lib_la_LIBADD = \
+   ta_abstract/libta_abstract.la \
+   ta_func/libta_func.la \
+-  ta_common/libta_common.la
++  ta_common/libta_common.la -lm
+ 
+ libta_lib_la_LDFLAGS = -version-info $(TALIB_LIBRARY_VERSION)
+ 
+diff --git a/src/Makefile.in b/src/Makefile.in
+index 489b44a..ef1f506 100644
+--- a/src/Makefile.in
```

Comment:
Is it your patch? if so, please make it -Np0

Reply:
not mine, took this one from gentoo

Reply:
i took it because of the -lm, but seem it's finally not needed.

Reply:
builds break without the patch ...